scripts/make_fake_pm_data.js

```diff
@@ -0,0 +1,134 @@
+const async = require("async");
+const fs = require("fs");
+const _ = require("underscore");
+const json2csv = require("json2csv");
+const argv = process.argv.slice(2);
+const BUILDINGS = require("./" + argv[0]);
+
+function randomNumber(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+}
+
+function randomNumberOfStandardDeviations(min, max) {
+    return ((Math.random() * (max - min + 1) + min))
```

Comment:
Is this `randomFloat()`? Maybe `randomInt` relies on it, or `randomInt` goes away and we just do a `Math.floor` where needed. The main point of this function is just to not have to think about how to get a ranged random, which is the boring but easy to mess-up part.